core/components/form-fields/shared/parse-fields.ts

```diff
@@ -225,24 +224,7 @@ export const parseRegisterCustomerFormData = (registerFormData: FormData): unkno
         });
       }
 
-      if (sections.includes('address')) {
-        parsedData.address[key] = value;
-      }
-
-      if (sections.some((section) => section.startsWith('custom_address'))) {
```

Comment:
Have you tested what happens if a Merchant decides to add custom address form field in CP?

Reply:
@bc-alexsaiannyi yes, custom fields added in CP are not displayed in the create account form. I have additionally added a video recording of this case.